packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -205,6 +206,7 @@ export type OutgoingPaymentEventType =
 export interface OutgoingPaymentResponse {
   id: string
   walletAddressId: string
+  client?: string
```

Comment:
Why make that optional?

Reply:
I assumed that not all implementations would require the `client` to be set, since the `client` is optional for the `outgoingPayments` table.

https://github.com/interledger/rafiki/blob/4e3894b22d5da7b85cfd4751cbb5369419f28021/packages/backend/migrations/20221129213751_create_outgoing_payments_table.js#L12

Reply:
Ok, there is probably no backwards compatible way to make that not nullable.

Reply:
Well, if it's created via the Admin API, this Rafiki is the client

Reply:
@sabineschaller if we always want to populate the `client` field including on Admin API requests (since we do not currently), then that is worth a separate issue (if not story) altogether